fhir-examples/src/main/resources/json/ibm/basic/BasicDate.json

```diff
@@ -98,7 +98,7 @@
         "_start": {
           "extension": [{
             "url": "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
-            "valueString": "masked"
+            "valueCode": "masked"
```

Comment:
is this fix needed for any tests, or just a general cleanup thing?   just checking since we currently point to the released fhir-examples.

Reply:
I have to update, otherwise I have to exclude it when I post to the server, which rejects it as invalid (no-value) data-absent-reason